content/chapter-parsers/opentelemetry/_index.md

```diff
@@ -27,3 +27,17 @@ String, bool, int64, double, and bytes values are mapped to their respective {{%
 The mapping of AnyValue type fields is limited.
 
 `ArrayValue` and `KeyValueList` types are stored serialized with protobuf type. Note that `protobuf` and `bytes` types are only available, unless explicitly type cast. For example, `bytes(${.otel.log.span_id})`. When using template functions, use `--include-bytes`, for example, `$(format-json .otel.* --include-bytes`. In the case of `$(format-json)`, the content is base64-encoded into the bytes content.
+
+## Options
+
+## set-hostname()
+
+|           |                              |
+| --------- | ---------------------------- |
+| Synopsis: | `yes | no ` |
+| Default: | `yes` |
+
+Available in {{< product >}} 4.8 and later.
+
+*Description:* If set to yes, the parser extracts the `host.name` attribute if available in the message. Otherwise, it leaves the [HOST](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host/) field as-is.
+If you're receiving messages using the [`syslog-ng-otlp()` source]({{< relref "/chapter-sources/source-syslog-ng-otlp/_index.md" >}}), the `opentelemetry()` parser will return the original hostname regardless of the where the message originated (OTLP, syslog, and so on).
```

Comment:
I would remove this part, as the `opentelemetry()` parser completely skips messages coming from `syslog-ng-otlp()`.